CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.42.1] - 2022-03-21
+### Added
```

Comment:
new feature. Minor version bump is prefered.

Reply:
sure!

Reply:
Can you approve this PR and #900 So that we only depend on starbase approval? 

This requested review from you because of import order change by formatting on geospatial files so starbase review would suffice for the rest of this PR. 🙏